specs/DesignDiscussions/InputLabels.md

```diff
@@ -115,55 +112,16 @@ The basic addition that we are going to make to the schema is to add a label pro
-
-There is another concern, however, which must be balanced against the desire to provide card authors with the ability to format their labels. Hosts may have label formatting in their app which they want to apply consistently to cards they are displaying. This applies particularly to how they mark fields as required or optional (per spec at [InputValidation.md](https://github.com/microsoft/AdaptiveCards/blob/master/specs/DesignDiscussions/InputValidation.md)). While by default we will mark required inputs with a `*`, we may want to allow hosts more control. Perhaps some apps wish to show labels for required inputs in bold or italics. If we allows hosts to specify formatting based on what is required, however, and we allow authors to specify formatting via a TextBlock or RichTextBlock, we can easily run into cases where the formatting comes into conflict.
-
-This conflict speaks to a general tension inherent in the design of Adaptive Cards. There's always going to be a trade off between giving the author more power, and letting the host control the experience for their users. We need to determine what balance we want to strike in this case.
-
-In order to gather more data to help us balance the needs of the card authors and hosts, we sent a survey to customers who have expressed an interest in improving out input story. The survey sent included 10 questions including the topics of input validation and input labels, for this spec we will only review the 4 questions that focus on the former topic. The results for this questions were used to leverage on some of the decisions taken on this spec document and can be found in the [Appendix](Appendix).
-
-### Default rendering for labels
-
-Reviewing the results from Q8 of the survey, we can see that most voters usually have a good experience with the default rendering of a TextBlock for rendering a label. From this result we can decide that string label rendering should be performed as if we were rendering a default TextBlock.
-
-This would allow authors to emulate writing a simple TextBlock element by simply defining the label as a string, making the string version look like this
-```json
-{
-	"type": "Input.Text",
-	"id": "textInput",
-	"label": "Label for Input"
-}
-```
-
-While the same rendering experience using a TextBlock would look like this
-```json
-{
-	"type": "Input.Text",
-	"id": "textInput",
-	"label": {
-		"type": "TextBlock",
-		"text": "Label for Input"
-	}
-}
-```
-
-Both of the afore mentioned cards would produce the same rendered result that would look like  this:
- 
-![img](assets/InputLabels/LabelDefault.PNG)
+The label should be rendered above the input as if the card author had used a TextBlock to label their input.
```

Comment:
>label should be rendered a [](start = 4, length = 26)

Does this need to be in its own "Renderer" sub-section? Is this the default behavior or the only behavior for hosts at this point? Can hosts customize this layout?

Reply:
Sure, i put a section header here. There's no position formatting at this time (all the available host formatting is described in the host formatting section) but the discussion of possible positioning changes still exists in the Appendix.

---
In reply to: [454691954](https://github.com/microsoft/AdaptiveCards/pull/4363#discussion_r454691954) [](ancestors = 454691954)